client/app/bundles/ConifyClient/startup/App.jsx

```diff
@@ -10,7 +10,7 @@ import HelloWorld from '../containers/HelloWorld';
 // This code here binds your smart component to the redux store.
 // railsContext provides contextual information especially useful for server rendering, such as
 // knowing the locale. See the React on Rails documentation for more info on the railsContext
-const App = (props, _railsContext) => {
+const App = (props/*, _railsContext*/) => {
```

Comment:
Even though we probably won't use it, I don't see any harm in keeping `_railsContext` so that data (such as user info) can potentially be pushed into the store before render from the server.